store/src/java/com/zimbra/cs/util/ProxyConfGen.java

```diff
@@ -1890,6 +1921,30 @@ public void update() {
     }
 }
 
+class WebStrictServerName extends WebEnablerVar {
+
+    public WebStrictServerName() {
+        super("web.strict.servername", "#",
+                "Indicates whether the default server block is generated returning a default HTTP response to all unknown hostnames");
+    }
+
+    @Override
+    public String format(Object o)  {
+        boolean strictServerName = isStrictEnforcementEnabled();
```

Comment:
nit pick: since you are only calling this once, there seems to be no need for a local boolean variable